KubeArmor/core/kubeArmor.go

```diff
@@ -485,6 +485,27 @@ func KubeArmor() {
 					return
 				}
 			}
+		} else if strings.HasPrefix(dm.Node.ContainerRuntimeVersion, "cri-o") { // cri-o
+			sockFile := false
+
+			for _, candidate := range []string{"/var/run/crio/crio.sock"} {
+				if _, err := os.Stat(candidate); err == nil {
```

Comment:
The same `for` loop is used in `KubeArmor/core/crioHandler.go` as well .. It would be better to create a function for this for two reasons:
1. reduced code
2. it is better to check for env var CRI_SOCKET and use that as the preferred one (first element in the string array) if it exists.

Reply:
Will refactor :+1: